lib/stat_tracker.rb

```diff
@@ -14,4 +14,34 @@ def self.from_csv(locations)
     StatTracker.new(games, teams, game_teams)
   end
 
+  def percentage_home_wins
```

Comment:
Overall, these methods look great and definitely work.  I think long-term we're going to need to move the bulk of these methods to their respective classes and then just call on that method from within the StatTracker, but this looks good for now.